ParisBNF/abb/BNFabb106.xml

```diff
@@ -0,0 +1,234 @@
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="51r" to="58v"/> 
+                                        </item>  
+                                    </list>
+                                </collation>
+                                <condition key="good"/>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="24">   
+                                    <ab type="ruling" subtype="pattern" cert="medium">1A-1A-1A1A-1A1A/0-0/0-0/C</ab>
+                                    <ab type="ruling">Ruling is partly visible. </ab>
+                                    <ab type="pricking"> Pricking is partly visible.</ab>
+                                    <ab type="ruling" cert="unknown"> The upper line is written above the ruling. </ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">19th-century script</seg> 
+                                <desc> <persName
+                                    ref="PRS1071dAbbadi">A. d'Abbadie</persName> describes the handwriting as <foreign xml:lang="fr">'fine'</foreign>.    
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                            <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF">Library</placeName> glued onto the front board.
+                                </desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF">Library</placeName> glued onto the inner side of the front board.
+                                    </desc>
+                                </item>
+                            </list>
+                            </additions>
+                        <bindingDesc>
+                            <binding contemporary="Ethiopian">
+                                <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>  
+                                <decoNote xml:id="b2" type="Cover">Leather quarter cover.</decoNote>  
+                                <decoNote xml:id="b3" type="SewingStations" cert="medium">4</decoNote>  
+                                <decoNote xml:id="b4" type="SewingStations" cert="medium">Two pairs of sewing stations.</decoNote>  
```

Comment:
b3 is correct, `type ``SewingStations `may only take a numeric value, b4 should have no type https://betamasaheft.eu/Guidelines/?id=bindingDescription see example 2 and 3